types/bmapgl/bmapgl.service.d.ts

```diff
@@ -324,7 +324,7 @@ declare namespace BMapGL {
     }
     interface AutocompleteResultPoi {
         province: string;
-        City: string;            // wtf
+        city: string;            // wtf
```

Comment:
Can you replace `// wtf` here with a note that says "the docs say City, but the actual code says city", or similar?

Seemingly someone was confused enough to have left the comment, which has turned out to be wrong, so it's good to note that in case someone comes back and is confused a second time.

Reply:
done~

I have changed the comment.